Input.cpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include "Engine.h"
+using namespace sf;
+
+
+void Engine::input()
+{
+	Event event;
+	updateMousePosition();
+
+	//Handle the player quitting
+	if (Keyboard::isKeyPressed(Keyboard::Escape))
+	{
+		m_Window.close();
+
+	}
+
+	////Handle the player starting the game
+	//if (Keyboard::isKeyPressed(Keyboard::Return))
+	//{m_Playing = true;} else {m_Playing = false;}
+
+	//Handle Mouse clicking
+	if (Mouse::isButtonPressed(Mouse::Left) && m_Playing == true)

```

Comment:
If you have a `bool` there is no need to compare it to `true`. Just write `&& m_playing` and that is it!